core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -135,3 +124,24 @@
     width: 220px;
   }
 </style>
+
+<link rel="stylesheet" href="/third_party/static/perfectscrollbar-0.6.16/css/perfect-scrollbar.css" />
+<style>
+  .ps-container:hover > .ps-scrollbar-y-rail:hover > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container:hover > .ps-scrollbar-y-rail:hover {
+    background-color: #c5e1a5;
+  }
+  .ps-container.ps-in-scrolling.ps-y > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+  .ps-container.ps-in-scrolling.ps-y:hover > .ps-scrollbar-y-rail > .ps-scrollbar-y {
+    background-color: #689F38;
+  }
+</style>
+
+<script src="/third_party/static/perfectscrollbar-0.6.16/js/perfect-scrollbar.js"></script>
```

Comment:
They really ought not to go in directives, I think. It might be a bit messy at the moment, but we should keep script and link tags to the controllers. If we can figure out a nice way to do a module system, then we could do this in a much cleaner way. Until then, I think it makes sense to just document at the top of the directive which third parties it depends on and expect the controller to include them when including the directive itself.